utils/parser.go

```diff
@@ -0,0 +1,46 @@
+package utils
```

Comment:
I don't like utils as a package name. I would rather do `filtering` or `filter` as package and then parsing and other functions will be in that package. utils means that I could not identify a domain object for this responsibility so lets use utils.

Reply:
This can be changed I think filters is fine

Reply:
Filters is not the right name if this is sorting criteria. It should be `order` or `ordering` package instead.

Reply:
Ok I didn't realize that utils is outside repository. This is a bad design. The db or repository related stuff should be inside repository package. And any filtering or sorting should be within postgres file since if we decide to have multiple repositories the implementation of repository will differ. Lets talk about this one.